syntax/javascript.vim

```diff
@@ -212,6 +211,10 @@ syntax region  jsCommentClass       contained start=/\/\*/ end=/\*\// contains=j
 syntax region  jsCommentMisc        contained start=/\/\// end=/$/    contains=jsCommentTodo,@Spell skipwhite skipempty nextgroup=jsBlock extend keepend
 syntax region  jsCommentMisc        contained start=/\/\*/ end=/\*\// contains=jsCommentTodo,@Spell skipwhite skipempty nextgroup=jsBlock fold extend keepend
 
+" Decorators
+syntax match   jsDecorator                    "^\s*@" nextgroup=jsDecoratorFunction
+syntax match   jsDecoratorFunction  contained "[a-zA-Z_][a-zA-Z0-9_.]*" nextgroup=jsParenDecorator
```

Comment:
I know this was from how it was previously, but while you are fixing this, might as well convert these to regexes instead of leaving them as strings